src/app/portlets/dot-edit-content/services/html/dot-dom-html-util.service.ts

```diff
@@ -46,6 +46,23 @@ export class DotDOMHtmlUtilService {
         return script;
     }
 
+    public getButtomHTML(label: string, className: string, dataset: {[key: string]: string}): string {
```

Comment:
Add a todo to a refactor for this, there is a way better way of doing this but I don't know by memory, have to research.